perllib/Integrations/SCP.pm

```diff
@@ -38,6 +72,7 @@ sub call {
         }),
         make_soap_structure_with_attr(@params),
     );
+    $self->log($res);
```

Comment:
This is quite a verbose thing give it's the entire SOM object, but I guess don't want to try and do too much in case it's something else and we miss the response.